test/unit/object/test_api_json_object.py

```diff
@@ -0,0 +1,3 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
```

Comment:
I would add a simple test in here. Just verify that if you instantiate this with a `dict`, that it has the same length, and is equal to the original dict.